CHANGELOG.md

```diff
@@ -9,17 +9,16 @@ _Feb 3, 2022_
 
 A big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
 
-- 🎁 Add support for master/detail (#3387) @m4theushw
+- 🎁 Add support for [master/detail](https://mui.com/components/data-grid/group-pivot/#master-detail) (#3387) @m4theushw
 
-  TODO: Add description + example
+  <img src="https://user-images.githubusercontent.com/42154031/152379354-47120aac-2b37-4a90-b311-64b4522283b9.gif" width="814">
 
 - 🌍 Add Danish (daDK) locale (#3800) @kasperfilstrup
 - 📚 Documentation improvements
 - 🐞 Bugfixes
 
 ### `@mui/x-data-grid@v5.5.0` / `@mui/x-data-grid-pro@v5.5.0`
 
-
```

Comment:
@alexfauquette The changelog script is adding two newlines.

Reply:
I prefer to say: "the prettier does not remove the two line breaks" ;)

The correction: #3833